deps/musl/default.nix

```diff
@@ -0,0 +1,49 @@
+{ nixpkgs ?
+  builtins.fetchTarball "https://github.com/NixOS/nixpkgs/archive/refs/tags/23.11.tar.gz"
```

Comment:
Tip: if you supply fetchTarball with `{ url = ...; sha256 = ...; }` instead of just a string, Nix will cache the download indefinitely. Currently it will cache for 1 hour or so, and then re-download next time `nix-build` is run.

Reply:
Nice, fixed!  Also this helped me find `nix-prefetch-url --unpack` which gives me the right sha, plus it seems to install the tarball into the store, which is exactly what I would have hoped for.